server/routes/todos.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const {
+  addTodos,
+  getTodos,
+  getTodoDetail,
+  editTodos,
+  deleteTodos,
+  changeCompleted,
+} = require("../controller/todoController");
+
+router.use(express.json());
+
+router.post("/", addTodos);
+router.get("/", getTodos);
+router.get("/:id", getTodoDetail);
+router.put("/:id", editTodos);
+router.delete("/:id", deleteTodos);
+router.put("/completed/:id", changeCompleted);
```

Comment:
같은 메소드끼리 이렇게 뭉쳐놓으면 좋을 것 같네요!
```suggestion
router.put("/:id", editTodos);
router.put("/completed/:id", changeCompleted);
router.delete("/:id", deleteTodos);
```